AppCreationScripts/apps.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an Android app",
+    "Level": 400,
```

Comment:
Is it really a 400? or a 200?

Reply:
Definetly not 400. Probably 100, maybe 200.